llvm/tools/llvm-exegesis/lib/SnippetFile.cpp

```diff
@@ -207,14 +208,16 @@ class BenchmarkCodeStreamer : public MCStreamer, public AsmCommentConsumer {
                     Align ByteAlignment, SMLoc Loc) override {}
 
   unsigned findRegisterByName(const StringRef RegName) const {
+    if (unsigned Reg = Target.findRegisterByName(RegName))
+      return Reg;
```

Comment:
Why is this just tacked on here? If there's registers missing, they should probably be added to the `RegNameToRegNo` map by modifying `LlvmState::createRegNameToRegNoMapping`. If this works better and covers all/more cases though, it might be something we want to do cleanup on, but that can probably be done in a separate PR along with the additional cleanup.

Reply:
That's a good idea, thanks for the suggestion. But I propose to make this in a separate pull request, since it does not provide functionality now.

Reply:
Could this actually be pulled out?

Reply:
Addressed, deleted in this PR